app/src/androidTest/java/com/example/android/courtcounter/robot/CounterTestRobot.kt

```diff
@@ -0,0 +1,26 @@
+package com.example.android.courtcounter.robot
+
+import androidx.test.core.app.ActivityScenario
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.assertion.ViewAssertions
+import androidx.test.espresso.matcher.ViewMatchers
+import androidx.test.espresso.matcher.ViewMatchers.withText
+import com.example.android.courtcounter.MainActivity
+import com.example.android.courtcounter.R
+import org.hamcrest.Matchers.allOf
+
+class CounterTestRobot {
+
+    val TEAM_A_SCORE = "0"
+
+    fun launchApp() {
+
+        ActivityScenario.launch(MainActivity::class.java)
+    }
+
+    fun checkTeamAPointsIsDisplayed() {
+
+        onView(allOf(ViewMatchers.withId(R.id.team_a_score), withText(TEAM_A_SCORE)))
```

Comment:
You might notice(particularly if you are copying/pasting code) imports will be added back in, as we have with ViewMatcher. Whether or not you streamline this is up to you. Just something to keep an eye on